acl_types.go

```diff
@@ -40,3 +40,15 @@ const (
 	AclResourceCluster         AclResourceType = 4
 	AclResourceTransactionalID AclResourceType = 5
 )
+
+type AclResourcePatternType int
+
+// ref: https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/resource/PatternType.java
+
+const (
```

Comment:
another way(idiomatic) to do it.

```suggestion
const (
	AclPatternUnknown  AclResourcePatternType = iota
	AclPatternAny      
	AclPatternMatch    
	AclPatternLiteral  
	AclPatternPrefixed 
	)
```

Reply:
TIL, thanks